lib/rubocop/cop/rspec/overwriting_setup.rb

```diff
@@ -44,18 +42,25 @@ def on_block(node)
 
         def find_duplicates(node)
           setup_expressions = Set.new
-          node.each_child_node do |child|
-            next unless setup?(child)
+          node.each_child_node(:block) do |child|
+            next unless common_setup?(child)
 
             name = if child.send_node.arguments?
-                     child.send_node.first_argument.value
+                     child.send_node.first_argument.to_a.first.to_sym
```

Comment:
Related to my question below, if you check that it's a `basic_literal?` first, can you just use `.value` as before?

Reply:
Nope, str doesn’t provide value

Reply:
`child.send_node.first_argument.to_a.first.to_sym` – I don’t understand this line. Would it be possible to use a beautifully named local variable or two, to make it more readable?